cellprofiler/modules/imagemath.py

```diff
@@ -404,7 +404,7 @@ def run(self, workspace):
             if opval == O_AVERAGE:
                 output_pixel_data /= sum(image_factors)
         elif opval == O_INVERT:
-            output_pixel_data = output_pixel_data.max() - output_pixel_data
+            output_pixel_data = 1.0 - output_pixel_data
```

Comment:
Nice. However, would you mind using `skimage.utils.invert`? It is a little safer (it computes the maximum dtype value) and can deal with binary images.